matchzoo/metrics.py

```diff
@@ -1,190 +1,210 @@
 """Evaluation metrics for information retrieval."""
 
+import math
+import random
 import numpy as np
+import typing
 
 
-def mean_reciprocal_rank(rs: list) -> float:
-    """
-    Calculate reciprocal of the rank of the first relevant item.
+def _to_list(x: typing.Any) -> list:
+    """Transfer the inputs to list."""
+    if isinstance(x, list):
```

Comment:
this function is useless if you clearly define x to be list or array-like objects in docstring